packages/babel-parser/src/tokenizer/index.js

```diff
@@ -917,6 +921,13 @@ export default class Tokenizer extends LocationParser {
             this.raise(this.state.pos, "Invalid or unexpected token");
           }
 
+          if (!allowNumSeparator) {
+            this.raise(
+              this.state.pos,
+              "Numeric separators are not allowed inside unicode escape sequences",
```

Comment:
The Numeric separators are also not allowed in hex escape sequences:
```js
var c = \x1_0
```

Reply:
Sure.